### PR TITLE
Refactor web initialization to utilize only asab default container.

### PR DESCRIPTION
**Pull request by PremyslCerny**

### CODE REVIEW

**Comment by PremyslCerny** [comment]
@ateska @byewokko This is BSPUMP adjustment to the latest ASAB default API approach.

File: bspump/web/__init__.py [hunk 1]
```diff
@@ -191,30 +195,11 @@ async def manifest(request):
...
-	# 	container.WebApp,
-	# 	root='/',
-	# 	path=os.path.join(os.path.dirname(__file__), "static"),
-	# 	index="app.html")
+def register_bspump_endpoints(container):
 
 	# Add routes
 	# LEGACY
-	# TODO: Remove this after September 2020
+	# TODO: Remove this after September 2021
```
Comment (ateska) [inline bspump/web/__init__.py:202]: We should remove this ;-)

File: bspump/web/__init__.py [hunk 2]
```diff
@@ -191,30 +195,11 @@ async def manifest(request):
...
-	# container = asab.web.WebContainer(websvc, 'asab:web', config={"listen": listen})
-
-	# Add web app
-	# TODO: The following app should be reviewed
-	# asab.web.StaticDirProvider(
-	# 	container.WebApp,
-	# 	root='/',
-	# 	path=os.path.join(os.path.dirname(__file__), "static"),
-	# 	index="app.html")
+def register_bspump_endpoints(container):
```
Comment (ateska) [inline bspump/web/__init__.py:198]: `initialize_web()`

**Review by ateska — CHANGES_REQUESTED** [review]
Smaller remarks, then mergeable.

**Pull request merged**